subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -22,44 +23,44 @@
 /**
  * BEWARE! This plugin is in incubating state, so its API may change in the future!
  * The plugin applies following plugins:
- *
+ * <p>
```

Comment:
What are these formatting changes for? It doesn't seems like an improvement, especially removing spaces/tabs from the beginnings of some lines

Reply:
Autoformatting unfortunately. 
I will rollback to the previous formatting. 
Sorry for that.